internal/server/data/group.go

```diff
@@ -16,15 +16,16 @@ func GetGroup(db *gorm.DB, selectors ...SelectorFunc) (*models.Group, error) {
 }
 
 func ListGroups(db *gorm.DB, selectors ...SelectorFunc) ([]models.Group, error) {
+	db = db.Order("name ASC")
```

Comment:
Noticed some test flakes because of a lack of sort order, so added this, which matches the order we use for `ListIdentities`